src/effects/NoiseReduction.cpp

```diff
@@ -74,7 +74,7 @@
 
 // SPECTRAL_SELECTION not to affect this effect for now, as there might be no indication that it does.
 // [Discussed and agreed for v2.1 by Steve, Paul, Bill].
-#undef EXPERIMENTAL_SPECTRAL_EDITING
+#undef SPECTRAL_EDIT_NOISE_REDUCTION
```

Comment:
This “obvious” idea from 2014 was vetoed by the old team and I forgot it.  But it remains latent.

You don’t yet see part 2 of this contribution.  I also want to rewrite the un-enabled experimentals with C++17 `if constexpr` instead of old `#ifdef` and that would include this idea.  That rewrite would let the compiler check those branches too so that they remain buildable while other parts of Audacity evolve.

I’m waiting first for acceptance of the easier rewrites in this PR.

Reply:
And so the file Experimental.cmake will be eliminated, and the switching-on of other experiments will also not require recompiling everything as now.